asteroid/data/avspeech_dataset.py

```diff
@@ -36,16 +36,16 @@ def get_frames(video):
 class Signal:
     """This class holds the video frames and the audio signal.
 
-        Args:
-            video_path (str,Path): Path to video (mp4).
-            audio_path (str,Path): Path to audio (wav).
-            embed_dir (str,Path): Path to directory that stores embeddings.
-            sr (int): sampling rate of audio.
-            video_start_length: video part no. [1]
-            fps (int): fps of video.
-            signal_len (int): length of the signal
-
-        .. note:: each video consists of multiple parts which consists of fps*signal_len frames.
+    Args:
```

Comment:
All black related commit should come in a single PR. 
I'm sorry to be rigid on that but this will make a cleaner history. 
Imagine if we introduced a bug with this PR and we have the revert everything and check what made the bug, and redo all the formatting. Having separate PRs is better.